src/platform/library/include/platform/lib/memory.h

```diff
@@ -10,6 +10,8 @@
 #ifndef __PLATFORM_LIB_MEMORY_H__
 #define __PLATFORM_LIB_MEMORY_H__
 
+#define PLATFORM_DCACHE_ALIGN	sizeof(void *)
```

Comment:
Btw, what are we aligning here ? Are we aligning on data cache line size and wouldn't this be architecture or platform specific ?

Reply:
@lgirdwood Yes, this is platform specific, so that's why it's in the platform headers. Previously it was placed in the arch for host, which was invalid.